### PR TITLE
Charged Higgs To CS

### PR DESCRIPTION
**Pull request by sanuvarghese**
We have cards for  2 Charged higgs  processes (h+ -> cs~ and h- -> c~s) at NLO level. The subdirectories(HplusToCS and HminusToCS) contain the cards for a sample mass point(120 GeV) plus a mass changing script that generates the cards for other masspoints.

### CODE REVIEW

**Comment by Saptaparna** [comment]
What is the difference between the cards aside from the Hplus or Hminus? Please commit a template (with a script where appropriate changes will be made), so it is easier for us to evaluate the cards.

**Comment by sanuvarghese** [comment]
The difference between hplus and hminus was in the madspin card. I have now modified the script to generate hplus and hminus cards for all mass points. A sample  card(hplus to cs for mass 80 GeV) is also included.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ChargedHiggsToCS/ChargedHiggsToCS_cards.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+echo ${masses[*]}
+
+# get length of an array
+tLen=${#postfix[@]}
+for charge in ${charges[*]};do
+    sample=H${charge}ToCS
+    for mass in ${masses[*]}; do
+        echo generating cards $charge, for M = $mass GeV
+	madspin=H${charge}ToCS_M${mass}_madspin_card.dat
+	echo "ecset ms_dir ./madspingrid">$madspin
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ChargedHiggsToCS/ChargedHiggsToCS_cards.sh:18]: ecset -> is this a typo?

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ChargedHiggsToCS/HminusToCS/HminusToCS_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+#***********************************************************************
+  'lhapdf'    = pdlabel     ! PDF set
+$DEFAULT_PDF_SETS = lhaid
+$DEFAULT_PDF_MEMBERS = reweight_PDF     ! if pdlabel=lhapdf, this is the lhapdf number
+#***********************************************************************
+# Include the NLO Monte Carlo subtr. terms for the following parton    *
+# shower (HERWIG6 | HERWIGPP | PYTHIA6Q | PYTHIA6PT | PYTHIA8)         *
+# WARNING: PYTHIA6PT works only for processes without FSR!!!!          *
+#***********************************************************************
+  HERWIG6   = parton_shower
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ChargedHiggsToCS/HminusToCS/HminusToCS_run_card.dat:75]: Do you use Herwig for showering? If not, this should be changed to PYTHIA8 (If you use Pythia)

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ChargedHiggsToCS/HminusToCS/HminusToCS_run_card.dat [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+            ! booleans of equal length to dynamical_scale_choice to
+            ! specify for which choice to include scale dependence.
+ False = reweight_PDF  ! Reweight to get PDF uncertainty. Should be a
+            ! list booleans of equal length to lhaid to specify for
+            !  which PDF set to include the uncertainties.
+#***********************************************************************
+# Store reweight information in the LHE file for off-line model-       *
+# parameter reweighting at NLO+PS accuracy                             *
+#***********************************************************************
+ False = store_rwgt_info ! Store info for reweighting in LHE file
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ChargedHiggsToCS/HminusToCS/HminusToCS_run_card.dat:110]: This should be turned on, True

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ChargedHiggsToCS/ChargedHiggsToCS_cards.sh (hunk 1 above)
Comment (sanuvarghese) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ChargedHiggsToCS/ChargedHiggsToCS_cards.sh]: Yeah. Corrected them now.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ChargedHiggsToCS/HminusToCS/HminusToCS_run_card.dat (hunk 2 above)
Comment (sanuvarghese) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ChargedHiggsToCS/HminusToCS/HminusToCS_run_card.dat]: Thanks.Changed the line to Pythia8

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ChargedHiggsToCS/HminusToCS/HminusToCS_run_card.dat (hunk 3 above)
Comment (sanuvarghese) [reply bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ChargedHiggsToCS/HminusToCS/HminusToCS_run_card.dat]: Done

**Comment by sihyunjeon** [comment]
Cards look fine to me @agrohsje

**Comment by sanuvarghese** [comment]
reminder @agrohsje

**Comment by sanuvarghese** [comment]
any updates? @sihyunjeon @agrohsje

**Comment by agrohsje** [comment]
looks good. thanks. will merge.

**Pull request merged**